uiplib/GUI.py

```diff
@@ -211,11 +211,23 @@ def set_wallpaper(self):
         image = self.images[self.index]
         change_background(image)
 
-    def enable_subreddit(self, mainFrame):
-        sub_label = Label(mainFrame, text="Enter Subreddits")
-        sub_label.grid(row=1, column=2, padx=0, pady=0)
-        self.sub_entry = Text(mainFrame, height=10, width=20)
-        self.sub_entry.grid(row=2, column=2, padx=10, pady=10, sticky=W)
+    def toggle_subreddit(self, mainFrame):
+        if self.reddit.get() == True:
+            try:
+                self.sub_label.grid()
```

Comment:
nice :D can you confirm that just calling grid places them at the exact attributes? a simple yes would suffice :D